CharacterCreation/02_Classes.txt

```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
 * Spotting
 * Survival
 
+== Hunter's Dog ==
+
+    When your character ascends to level 3, you receive an attack dog. Maybe 
+you found the dog along your last mission, or maybe a dog was issued to you when
+you returned to home base (at the GM's choice). Every time you level up after
+you have your dog, your dog levels up as well as defined in the 
+"Class-Specific Documentation/Hunter's_Dog.txt" document. For simplicity, Dogs 
+start at level 3 so that they are always the same level as their owners. If your
+dog is killed in action, you do not get to roll another one. If you and your dog
+are separated and level up separately, your dog immediately levels up to match 
+your level when you are reunited. 
+
+    Your new dog is well-trained and responds to a variety of commands. Your dog 
+listens to you and does his best to make his or her owner happy, but is not 
+controlled by the player. Instead, the owner character commands the dog, and the
+GM controls how the dog responds. 
+
+Dog Personalities:
+
+
+
+*  Loyal
+*  Playful
+*  Mischevious
+*  Protective
+*  Curious
+*  Aggressive
 
 == Class Feats ==
-	----------------------
-	Hunter Vision - Lvl/2 uses per day
-	Your breathing calms, as time appears to slow down. Your vision turns into 
-	myriad shades of grey. You see a faint path of color representing the trail of 
-	your quarry. +20% to a tracking check, Or take 3 turns to pass the check. 
-	Passively, gain the ability to sense where your Marked Quarry is, within
-	10 meters of you.
-	|
-	|
-	V
-	The Trail becomes bright against a monochrome background of edged shades of 
-	grey. You can see marks and paths that others simply cannot. +50% to a tracking 
-	check, or maximum 2 turns to pass. Passively, gain the ability to sense where your
-	Marked Quarry is, within 20 meters of you.
-	----------------------
-	Highlight Quarry - 
-	If your Quarry is marked, for 1 action, you can activate a Marked Quarry's nanites to make them 
-	more visible. All damage dealt to this target is increased by 10. Two uses per encounter, one use
-	per Quarry. Maximum of Lvl number uses per day. Lasts for the duration of the encounter.
-	At level 10, all damage dealt to this target is increased by 20 (instead of 10).
-	----------------------
-	They Never Saw it Coming -
-	If your Quarry has not seen you for at least 1 turn, your next attack against them does +30 bonus damage.
-	Triggerable once per Quarry.
-	Increases to +50 bonus damage at level 10.
-======================================
+
+Tracker:
+	As an Action, you can study the position and disposition of a target that
+you can see, and indicate it to your Party Members that can communicate with 
+you, either by being on comms with you (see the headset and similar items in
+the "Items/07_Items.txt" document) or that are within 5m of you. Party Members
```

Comment:
I assume that the within 5m is to indicate secrecy. It isn't directly stated that this is being done quietly, and so the 5m seems like a hard limit on the hearing range of your team. I don't think that was the intention.
Also, could this be indicated at a longer range quietly by using hand-signals?

Reply:
I have decided not to change this one. Describing quickly and accurately where an enemy is and what their stance is requires both words and hand signals at the same time. Doing it over gunfire requires that your ally be close. 
Mechanically, this also helps lend itself to the design goal of further cementing the Hunter class as a pack hunter, who needs to be close to at least one ally to be in their pack.